internal/utils/csi.go

```diff
@@ -337,10 +337,26 @@ var DriverNamespaceEnvVar = corev1.EnvVar{
 	},
 }
 
-// CSI Addons container port definition
-var CsiAddonsContainerPort = corev1.ContainerPort{
+// CSI Addons container port definitions
+var ControllerPluginCsiAddonsContainerBasePort = corev1.ContainerPort{
 	ContainerPort: int32(9070),
 }
+var ControllerPluginCsiAddonsContainerPort = ControllerPluginCsiAddonsContainerBasePort
+var ControllerPluginCsiAddonsContainerRbdPort = ControllerPluginCsiAddonsContainerBasePort
+var ControllerPluginCsiAddonsContainerCephFsPort = corev1.ContainerPort{
+	ContainerPort: int32(9080),
+}
+
+var NodePluginCsiAddonsContainerBasePort = corev1.ContainerPort{
+	ContainerPort: int32(9071),
+}
+var NodePluginCsiAddonsContainerPort = NodePluginCsiAddonsContainerBasePort
+
+func ContainerPortArg(port corev1.ContainerPort) string {
+
+	return fmt.Sprintf("--controller-port=%d", port.ContainerPort)
+
+}
```

Comment:
```suggestion
var ControllerPluginCsiAddonsContainerRbdPort = corev1.ContainerPort{
	ContainerPort: int32(9070),
}

var ControllerPluginCsiAddonsContainerCephFsPort = corev1.ContainerPort{
	ContainerPort: int32(9080),
}

var NodePluginCsiAddonsContainerPort = corev1.ContainerPort{
	ContainerPort: int32(9071),
}

func ControllerPortArg(port corev1.ContainerPort) string {

	return fmt.Sprintf("--controller-port=%d", port.ContainerPort)

}
```

We dont need extra variables we can have only 3 vars